docs/quickstart.md

```diff
@@ -0,0 +1,428 @@
+```
+
+```bash Warning
+[2] petstore.yaml:14:7 at #/paths/~1pets/get
+
+Operation object should contain `operationId` field.
+
+12 | /pets:
+13 |   get:
+14 |     summary: List all pets
+15 |     operationIdentifier: listPets
+ … |     < 27 more lines >
+43 |               $ref: "#/components/schemas/Error"
+44 |   post:
+45 |     summary: Create a pet
+
+Warning was generated by the operation-operationId rule.
+```
+
+Why did you get it? When you execute the `lint` command, OpenAPI CLI runs a large number of *rule-based* tests to verify that your definition file is correct. A rule is an atomic piece of code to check a particular part of the definition file. OpenAPI CLI ships a set of built-in rules but you can also define your own. However, to check something for validity, you must have a source of truth for what is considered valid or not. This source is the [OpenAPI Specification](https://spec.openapis.org/oas/latest.html). When you get an error, it means that one or more rules detected inconsistency of your definition file with the OpenAPI Specification.
+
+Let's inspect the `lint` command output. As you can see, there is 1 error (point [1]) and 1 warning (point [2]). The information we (and you) are interested in is the following:
+
+For the error:
+* [1] petstore.yaml:15:7 (the error is somewhere at line 15 near the 7th symbol)
+* Property `operationIdentifier` is not expected here (unexpected property)
+* Error was generated by the spec rule (a rule that detected the error)
+
+For the warning:
+* [1] petstore.yaml:14:7 (the error is somewhere at line 14 near the 7th symbol)
+* Operation object should contain `operationId` field (there should be a particular field here)
+* Warning was generated by the operation-operationId rule (a rule that detected the warning)
+
+You got that error because you've changed a property that is strictly defined in the specification. The linter has the [`spec`](./built-in-rules.md#spec) built-in rule that generates an error each time when it finds something that is not defined in the specification.
+
+The warning is a consequence of the error - when you changed the `operationId`, you also changed the object that this property is a part of - the [Operation object](https://spec.openapis.org/oas/latest.html#operation-object). Since the `operationId` is no longer there, you get a warning triggered by the corresponding built-in rule - [operation-operationId](./built-in-rules.md#operation-operationid).
+
+Let's make you definition valid again. If you look at the defined Field Names of the [Operation object](https://spec.openapis.org/oas/latest.html#operation-object) (after the warning description), you can quickly detect what's wrong with your definition file - there cannot be a Field Name with the `operationIdentifier` name, it is invalid. So, change it back to the `operationId` and you're done. In this way, you will fix both the warning and the error because the unexpected property `operationIdentifier` is gone.
```

Comment:
```suggestion
Let's make your definition valid again. Look at the defined Field Names of the [Operation object](https://spec.openapis.org/oas/latest.html#operation-object) (after the warning description). Detect what's wrong with your definition file - there cannot be a Field Name with the `operationIdentifier` name (it is invalid). Change it back to the `operationId` and you're done. You will fix both the warning and the error because the unexpected property `operationIdentifier` is gone.
```

Reply:
I attempted to streamline and shorten sentences.